.gitignore

```diff
@@ -82,3 +82,9 @@ cucumber-results/
 
 /dropbox
 public/source_maps
+
+## From Docker deploy
+Gemfile.local
+config/lti.yml
+config/resque.yml
```

Comment:
The docker script comes primarily from upstream the avalon project, and modifies these files. I think they are safe to ignore here. Alternative would be to use the "git update-index --assume-unchanged" trick, but I think this change makes things more explicit.